src/IncrementalChecker.ts

```diff
@@ -128,9 +133,26 @@ class IncrementalChecker {
     return endsWith(filePath, '.d.ts') || linterExclusions.some(matcher => matcher.match(filePath));
   }
 
+  /**
+   * Check for vue-loader in user's project node_modules.
+   */
+  hasVueLoader(): boolean {
+    if (this.vueLoaderResolved !== undefined) return this.vueLoaderResolved;
+    
+    try {
+      Resolve.sync('vue-loader', { basedir: this.basedir });
+      this.vueLoaderResolved = true;
+    } catch (e) {
+      // vue-loader doesn't exist, so return false.
+      this.vueLoaderResolved = false;
+    }
+
+    return this.vueLoaderResolved;
+  }
+
   nextIteration() {
     if (!this.watcher) {
-      this.watcher = new FilesWatcher(this.watchPaths, ['.ts', '.tsx']);
+      this.watcher = new FilesWatcher(this.watchPaths, ['.ts', '.tsx', '.vue']);
```

Comment:
Could including `vue` be conditional upon whether this is a `vue` program or not please?

Reply:
Does it hurt to always include `.vue`? It doesn't really bother the folks not on Vue.js. Otherwise it's another condition we'll need to make here for the watcher. Thoughts?

Reply:
2 things really:

1. It will make a marginal speed difference for non-Vue users. I'm philosophically opposed to that :smile:
2. It bothers me on an OCD level that Vue files would be always in the mix when they needn't be. This is more subjective I grant you but makes me twitch in unhappy ways. It is my suffering.

Reply:
Roger that, ok, let me see of a clean way to condition that...

Reply:
Thanks man!